### DISCUSSION TITLE
Stage Atmosphere Sound

### DISCUSSION THREAD

**Discussion by WF4123** [ikemen-engine/Ikemen-GO#905] [Feature Request] [answered]
Suggestion coming right up, for Stage Atmosphere Sound.

In this feature, we're just look around for some interesting stage system feature from it's like Tekken 4, The Last Blade, Capcom VS. SNK 2, etc.

[SNDDef]
snd = stage.snd

[SND Raining]
sndno = 700, 0
start = 0 (sample)
loop = 1

[SND Thunder]
sndno = 701, 0+(random%3)
start = 780 (sample)
looptime = 220 (sample)

**Comment by LasombraDemon** [comment, marked as answer]
A nice idea! But AttachedChar already can already pull this. It was made for it, honestly.

Imagine a cool ambience sound effect: get a bunch of sounds you wanna play. The ones that are short (an owl's yell, a wolf's howl) can be randomized and given a fadein/fadeout effect. Don't fadeout the thunder, of course. The others, keep as is.

Now put all that into an SND file.

Then make an AttachedChar that plays those bigger constant soundbites on loop, and does the small soundbites that fade in-out... At random.

Completely doable already and quite interesting, if you ask me. :)
